ibis/omniscidb/operations.py

```diff
@@ -289,6 +290,37 @@ def extract_field_formatter(translator, expr):
     return extract_field_formatter
 
 
+def _log_common(translator, arg, base=None):
```

Comment:
can use add a comment like ``# NUMERIC`` here before this section (are there other ops where this should go, e.g. sin, cos, IIRC we have a PR about this, but not merged)